app/javascript/packages/address-search/index.tsx

```diff
@@ -1,13 +1,15 @@
 import { snakeCase, formatLocations, transformKeys } from './utils';
 import AddressInput from './components/address-input';
 import AddressSearch from './components/address-search';
+import FullAddressSearch from './components/full-address-search';
 import InPersonLocations from './components/in-person-locations';
 import NoInPersonLocationsDisplay from './components/no-in-person-locations-display';
 import { requestUspsLocations } from './hooks/use-usps-locations';
 
 export {
   AddressInput,
   InPersonLocations,
+  FullAddressSearch,
   NoInPersonLocationsDisplay,
```

Comment:
Now that `FullAddressSearch` will be available to import, I'd like to suggest that we increase the version of `@18f/identity-address-search` so that we can get this published to npm. Maybe out of scope for this issue- maybe not- can we also published this to npm so it can be used? (You need access- but it is one command)

Reply:
Yep! My plan is to:
1. Merge this PR
2. Bump the minor version
3. Publish the new version

Let me know if that seems off to you

Reply:
In retrospect I should have bumped the minor version _in this PR_. Now I'm creating a separate PR to just bump the package version 😮‍💨